chia/cmds/init_funcs.py

```diff
@@ -381,6 +400,18 @@ def chia_init(
         fix_ssl(root_path)
     if should_check_keys:
         check_keys(root_path)
+    if experimental_v2_db:
+        config: Dict = load_config(root_path, "config.yaml")["full_node"]
+        db_path_replaced: str = config["database_path"].replace("CHALLENGE", config["selected_network"])
+        db_path = path_from_root(root_path, db_path_replaced)
+        mkdir(db_path.parent)
+        import sqlite3
+
+        with sqlite3.connect(db_path) as connection:
```

Comment:
Given the `experimental` naming, I'm assuming that this option won't exist for too long. I was wondering if there are concerns about blindly inserting a row without checking if a row already exists (maybe specifying an even newer (future) version). If the plan is to remove this code after a while then I think this is fine for now.

It looks like execution of this code is guarded by the earlier check for:
`if root_path.is_dir() and Path(root_path / "config" / "config.yaml").exists():`

So re-running `chia init --experimental-v2-db` several times shouldn't cause any trouble.

Reply:
I would think this should be removed once we make v2 the default. Also, this path is only executed when we don't already have a `.chia` folder, so it's guaranteed there won't be any tables.

The schema version can't be changed like this once tables have been created.